### PR TITLE
Added grouping to Postgres...SkuStep

### PR DESCRIPTION
**Pull request by neelip**
image

image

`Show pricing information...` is directed to https://aka.ms/AAcxhvm

### CODE REVIEW

**Comment by neelip** [comment]
@fiveisprime what do you think about the recommended group?

**Comment by fiveisprime** [comment]
Looks good!

File: src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts [hunk 1]
```diff
@@ -27,17 +25,28 @@ export class PostgresServerSkuStep extends AzureWizardPromptStep<IPostgresServer
             "selectPostgresSku",
             "Select the Postgres SKU and options."
         );
-        wizardContext.sku = (
-            await ext.ui.showQuickPick(this.getPicks(), { placeHolder })
-        ).data;
+        const pricingTiers: IAzureQuickPickItem<Sku | undefined>[] = await this.getPicks();
+        const moreInfo: IAzureQuickPickItem = { label: localize('ShowPricingCalculator', '$(link-external) Show pricing information...'), data: undefined }
```
Comment (ejizba) [inline src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts:29]: You can use `onPicked` like [here](https://github.com/microsoft/vscode-azuretools/blob/cc46259643436649b85f40cc9e843d1ef95d19a7/ui/src/wizard/StorageAccountListStep.ts#L148) to simplify this code. It should get rid of the need for a loop and also doesn't close & reopen the quick pick when the url is clicked

File: src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts [hunk 2]
```diff
@@ -27,17 +25,26 @@ export class PostgresServerSkuStep extends AzureWizardPromptStep<IPostgresServer
...
+            label: localize('ShowPricingCalculator', '$(link-external) Show pricing information...'),
+            onPicked: async () => {
+                await openUrl('https://aka.ms/AAcxhvm');
+            }, data: undefined
+        });
+
+        const pick: IAzureQuickPickItem<Sku | undefined> = await ext.ui.showQuickPick(pricingTiers, { placeHolder, suppressPersistence: true, enableGrouping: true });
+        if (pick.data) {
+            wizardContext.sku = pick.data;
+        }
```
Comment (ejizba) [inline src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts:39]: nit: this is what I usually do
```suggestion
        wizardContext.sku = (await ext.ui.showQuickPick(pricingTiers, { placeHolder, suppressPersistence: true, enableGrouping: true })).data;
```

**Review by ejizba — APPROVED** [review]

**Comment by tonybaloney** [comment]
Thanks @neelip, this is a great improvement

**Pull request merged**